src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,63 @@
 package core.basesyntax;
 
+import java.util.LinkedList;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private LinkedList<Entry<K, V>>[] buckets;
+    private int size;
+
```

Comment:
There is an empty line at the beginning of the class implementation. According to the checklist, avoid starting class or method implementations with empty lines.